test/data/devicegraphs/md_raid_lvm.xml

```diff
@@ -0,0 +1,277 @@
+<?xml version="1.0"?>
```

Comment:
At least in master I would expect to be able to define an LVM on top of a RAID using the yml format. And to be honest, it's much more human-compatible (and you could omit many unnecessary details.

Can you check if that's possible in this branch as well?

We even have a conversion tool